x-pack/plugins/actions/server/plugin.ts

```diff
@@ -220,22 +213,26 @@ export class ActionsPlugin implements Plugin<Promise<PluginSetupContract>, Plugi
 
     const usageCollection = plugins.usageCollection;
     if (usageCollection) {
-      initializeActionsTelemetry(
-        this.telemetryLogger,
-        plugins.taskManager,
-        core,
-        await this.kibanaIndex
+      registerActionsUsageCollector(
+        usageCollection,
+        core.getStartServices().then(([_, { taskManager }]) => taskManager)
       );
-
-      core.getStartServices().then(async ([, startPlugins]) => {
-        registerActionsUsageCollector(usageCollection, startPlugins.taskManager);
-      });
     }
 
-    core.http.registerRouteHandlerContext(
-      'actions',
-      this.createRouteHandlerContext(core, await this.kibanaIndex)
-    );
+    this.kibanaIndexConfig.subscribe((config) => {
```

Comment:
I noticed that for other uses, we actually only deal with the "first" update of `kibanaIndexConfig` via `kibanaIndexConfig.pipe(first()).toPromise()`, but not here.  

So it seems like if `this.kibanaIndexConfig` emits > 1 times per Kibana startup, we'd try to register the routes > 1 times.  The same pattern is used in alerts.  

Thinking perhaps we should just set `kibanaIndexConfig` to the following, for consistency, then we can remove the other `pipe(first())` usages.  Or perhaps I'm missing something.

```
initContext.config.legacy.globalConfig$.pipe(first())
```